scripts/android/cmake.sh

```diff
@@ -117,8 +117,8 @@ fi
 printn Y "Cleaning the folder... "
 print G "done."
 
-rm -rf .tmp || die "Failed to remove the temporary directory"
-mkdir .tmp || die "Failed to create the temporary directory"
```

Comment:
That was an Old thing we needed to to due to Qmake beeing wierd. 
We only need to remove android-build, so that on each compile the changes in `/android` get synced over. 
This should stop us pulling rust,sentry, etc everytime.

Reply:
Wait, but this is not in the PR anymore. Is that correct?

Reply:
If this comes back, better to link the issue it fixes https://mozilla-hub.atlassian.net/browse/VPN-3880

Reply:
oops- yes reverted too much of that file :D - will come back and i can link it.